### PR TITLE
feat: add xsl stylesheet to changelogs

### PR DESCRIPTION
**Pull request by psykzz**
## About the PR

modified changelog.xml to include an xsl stylesheet

## Why / Balance

better looks

## Technical details

replaced the default python xml with lxml which is a cpython binding.
It provides better support for xsl, by allowing us to specify the additional doctype header.

The formatting change was just `black` doing its thing, i can revert if needed.

## Media

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by moonheart08** [comment]
i didnt even know this was a thing you could do with xml
neat

File: Tools/actions_changelog_rss.py [hunk 1]
```diff
@@ -102,23 +101,40 @@ def main():
...
-            et.write(f, encoding="utf-8", xml_declaration=True)
-
-
-def create_feed(changelog: Any, previous_items: List[ET.Element]) -> Tuple[ET.Element, bool]:
+            et.write(
+                f,
+                encoding="utf-8",
+                xml_declaration=True,
+                # This ensures our stylesheet is loaded
+                doctype="<?xml-stylesheet type='text/xsl' href='./stylesheet.xsl'?>",
```
Comment (PJB3005) [inline Tools/actions_changelog_rss.py:109]: I don't see this parameter in the documentation: https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.ElementTree.write

**Review by PJB3005 — CHANGES_REQUESTED** [review]
Definitely revert the changes to the styling please.

File: Tools/actions_changelog_rss.py (hunk 1 above)
Comment (psykzz) [reply Tools/actions_changelog_rss.py:109]: It was the reason to replace it with lxml

Comment (PJB3005) [reply Tools/actions_changelog_rss.py:110]: Ah, I missed the change to lxml

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**